include/netuit/assign/AssignContiguously.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
run `./ci/generate_header_guards.sh` (but be sure to add whitespace at the top and at the end of `.hpp` files first)